src/Workspaces/CSharp/Portable/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -41,6 +46,11 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                         return null;
                     }
 
+                    if (currentToken.IsPattern())
+                    {
+                        return null;
```

Comment:
why?  this doesn't seem right to me.  We would want to adjust newlines.  We'd just all want that be overridden by a supression if it was on a single line to start with. But if this is spread over multiple lines, we would want this fixed up, right?